src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs

```diff
@@ -338,6 +354,12 @@ private DriveInfo ReportCommonDiskUsage(string dirType, string directory)
                 }
                 return true;
             }
+            catch (PathTooLongException)
```

Comment:
Let's meet up and chat about this.  Haven't actually updated buildtools in CoreFX yet, but this has been the state of every timed-out-on-this-step build we've seen (waiting on RoboCopy to return for hours)